packages/devtools_app/lib/src/app_size/app_size_table.dart

```diff
@@ -77,7 +77,8 @@ class _NameColumn extends TreeColumnData<TreemapNode> {
 }
 
 class _SizeColumn extends ColumnData<TreemapNode> {
-  _SizeColumn() : super('Size', alignment: ColumnAlignment.right);
+  _SizeColumn()
+      : super('Size', alignment: ColumnAlignment.right, fixedWidthPx: 100.0);
```

Comment:
we were overriding fields instead of using the existing parameter in the constructor. Cleaned that up so that we can make the constructor parameter required. Fyi @terrylucas.